packages/block-editor/src/components/block-rename/rename-control.js

```diff
@@ -54,9 +60,12 @@ export default function BlockRenameControl( { clientId } ) {
 		<>
 			<MenuItem
 				onClick={ () => {
-					setRenamingBlock( true );
+					if ( onRenameBlock ) {
+						onRenameBlock( clientId );
+					} else {
+						setRenamingBlock( true );
+					}
 				} }
-				aria-expanded={ renamingBlock }
```

Comment:
Since we're opening a dialog and we won't be able to access this element when the dialog is open, I don't see why we need this 🤔. I could be wrong though so could use a sanity check from @Mamaduka. 🙏